platform/bootstrap/bootstrap.yml

```diff
@@ -31,40 +104,44 @@ Parameters:
     - t2.large
     - m3.medium
     - m4.large
-    ConstraintDescription: Must be a valid EC2 HVM instance type
-    Default: m3.medium
-    Description: EC2 HVM instance type (t2.micro, m3.medium, etc)
-  InstanceType:
+    ConstraintDescription: Must be a valid EC2 HVM instance type.
+    Default: t2.micro
+    Description: EC2 HVM instance type (t2.micro, m3.medium, etc).
+  ManagerDiskSize:
+    Type: Number
+    Default: 20
+    MaxValue: 1024
+    MinValue: 20
+    Description: Size of Manager's ephemeral storage volume in GiB.
+  ManagerDiskType:
     Type: String
     AllowedValues:
+      - standard
+      - gp2
+    Default: standard
+    Description: Manager ephemeral storage volume type.
+  WorkerInstanceType:
+    Type: String
+    AllowedValues:
+    - t2.nano
```

Comment:
we should add more options: m4.xlarge, m4.2xlarge, c4.large, c4.xlarge, c4.2xlarge, c4.4xlarge, r4.large, r4.xlarge, r4.2xlarge, r4.4xlarge.